apps/discord-bot/src/core/CUGetReg.ts

```diff
@@ -52,10 +52,10 @@ export class CUGetReg extends Client {
   }
 
   get commands(): Map<string, ICommand> {
-    return this.commandList
+    return this._commandList
   }
```

Comment:
i meant `get commands()` should match the private variable name `_commands` not `_commandList`